package.json

```diff
@@ -14,7 +14,8 @@
     "discord.js": "^11.1.0",
     "dotenv": "^4.0.0",
     "lodash": "^4.17.4",
-    "lowdb": "^0.16.2"
+    "lowdb": "^0.16.2",
+    "nedb": "^1.8.0"

```

Comment:
I don't know if we want that... I mean at some point we gonna ditch one of the databases anyway... I just added it nedb this way so both are available and you can see how they compare...
Optional dependencies sound pretty sophisticated as it would require some kind of code flexibility as well... (no clue actually - that is just how I imagine how it would work) and that is quite a bit of struggle for something that will probably not kept how it is right now.